packages/@react-types/dialog/src/index.d.ts

```diff
@@ -45,15 +45,25 @@ export interface SpectrumDialogProps extends DOMProps, StyleProps {
 }
 
 export interface SpectrumAlertDialogProps extends DOMProps, StyleProps {
+  /** The [visual style](https://spectrum.adobe.com/page/dialog/#Options) of the AlertDialog.  */
   variant?: 'confirmation' | 'information' | 'destructive' | 'error' | 'warning'
+  /** The title of the AlertDialog. */
   title: string,
+  /** The contents of the AlertDialog. */
   children: ReactNode,
+  /** The label to display within the cancel button. */
   cancelLabel?: string,
+  /** The label to display within the confirm button. */
   primaryLabel?: string,
+  /** The label to display within the secondary button. */
   secondaryLabel?: string,
+  /** Whether the confrim button is disabled. */
   isConfirmDisabled?: boolean,
+  /** Handler that is called when the cancel button is pressed. */
   onCancel?: () => void,
+  /** Handler that is called when the confirm button is pressed. */
   onConfirm?: (button: 'primary' | 'secondary') => void,
+  /** Button to focus by default upon render. */
   autoFocusButton?: 'cancel' | 'primary' | 'secondary',
   allowsKeyboardConfirmation?: boolean, // triggers primary action
   isKeyboardCancelDisabled?: boolean // needed?
```

Comment:
Should I delete these props for now? They aren't currently in use but I dunno if there are plans to add them soon

Reply:
can you comment them out instead?